docs/src/css/_navbar.scss

```diff
@@ -49,15 +49,24 @@
     --ifm-dropdown-hover-background-color: var(--dwc-color-primary-alt);
   }
 
+  #webforj-version-badge {
+    background-color: #000;
```

Comment:
The black color works, but it feels a bit dull. I tried primary and other colors, but they draw too much attention. Don’t spend too much time on it, just see if there are better options.